MNT-22737: Handle Boundary Event Properly (#3947)

vrmoreira opened the pull request.

AnkurChauhan-131 committed:
MNT-22737: Handle Boundary Event Properly (#3947)


Co-authored-by: Vitor Moreira <vitor.moreira-v@alfresco.com>

dependabot[bot] committed:
build(deps): bump org.apache.maven.plugins:maven-resources-plugin (#4462)

Bumps [org.apache.maven.plugins:maven-resources-plugin](https://github.com/apache/maven-resources-plugin) from 3.3.0 to 3.3.1.
- [Release notes](https://github.com/apache/maven-resources-plugin/releases)
- [Commits](https://github.com/apache/maven-resources-plugin/compare/maven-resources-plugin-3.3.0...maven-resources-plugin-3.3.1)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-resources-plugin
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump org.apache.maven.plugins:maven-jar-plugin (#4461)

Bumps [org.apache.maven.plugins:maven-jar-plugin](https://github.com/apache/maven-jar-plugin) from 3.2.2 to 3.3.0.
- [Release notes](https://github.com/apache/maven-jar-plugin/releases)
- [Commits](https://github.com/apache/maven-jar-plugin/compare/maven-jar-plugin-3.2.2...maven-jar-plugin-3.3.0)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-jar-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump org.apache.maven.plugins:maven-clean-plugin (#4466)

Bumps [org.apache.maven.plugins:maven-clean-plugin](https://github.com/apache/maven-clean-plugin) from 3.2.0 to 3.3.2.
- [Release notes](https://github.com/apache/maven-clean-plugin/releases)
- [Commits](https://github.com/apache/maven-clean-plugin/compare/maven-clean-plugin-3.2.0...maven-clean-plugin-3.3.2)

---
updated-dependencies:
- dependency-name: org.apache.maven.plugins:maven-clean-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

igdianov committed:
AAE-16959 Add mysql upgrade scripts for identity link details (#4447)

balsarori reviewed (approved):
LGTM. Although, I recommend removing extra noise from the process BPMN XML such as diagram info and other unneeded extensions.

vrmoreira committed:
MNT-22737: XML file cleanup

vrmoreira committed:
Revert "MNT-22737: XML file cleanup"

This reverts commit 79fc9e4cdf3bce226de0f23ab7c00b26e145e99a.

alfresco-build committed:
Update to version 8.2.0-SNAPSHOT [skip ci]

dependabot[bot] committed:
build(deps): bump org.ow2.asm:asm-analysis from 9.3 to 9.6 (#4464)

Bumps org.ow2.asm:asm-analysis from 9.3 to 9.6.

---
updated-dependencies:
- dependency-name: org.ow2.asm:asm-analysis
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump org.owasp:dependency-check-maven from 7.1.2 to 8.4.3 (#4478)

Bumps [org.owasp:dependency-check-maven](https://github.com/jeremylong/DependencyCheck) from 7.1.2 to 8.4.3.
- [Release notes](https://github.com/jeremylong/DependencyCheck/releases)
- [Changelog](https://github.com/jeremylong/DependencyCheck/blob/main/CHANGELOG.md)
- [Commits](https://github.com/jeremylong/DependencyCheck/compare/v7.1.2...v8.4.3)

---
updated-dependencies:
- dependency-name: org.owasp:dependency-check-maven
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Giovanni007 committed:
AAE-18108 Use SHA for GitHub actions (#4485)

* AAE-18108 Add pre-checks job

* AAE-18108 Use pinned SHA

* AAE-18108 update-dependabot.yml

dependabot[bot] committed:
build(deps): bump Alfresco/alfresco-build-tools from 3.0.2 to 4.0.0 (#4488)

Bumps [Alfresco/alfresco-build-tools](https://github.com/alfresco/alfresco-build-tools) from 3.0.2 to 4.0.0.
- [Release notes](https://github.com/alfresco/alfresco-build-tools/releases)
- [Commits](https://github.com/alfresco/alfresco-build-tools/compare/v3.0.2...v4.0.0)

---
updated-dependencies:
- dependency-name: Alfresco/alfresco-build-tools
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

vrmoreira committed:
MNT-23915: errors thrown in called activities get caught by parent process (#4490)

* MNT-22737: Handle Boundary Event Properly (#3947)
Co-authored-by: Vitor Moreira <vitor.moreira-v@alfresco.com>

* MNT-23915: errors thrown in called activities get caught by parent process (#4474)
* Only checks if it's a called activity if there's no catch been executed
Co-authored-by: Bassam Al-Sarori <2126270+balsarori@users.noreply.github.com>
(cherry picked from commit 3c2e714f96e8949599835c703a4fd5d92b8bc3b7)

Co-authored-by: Ankur Chauhan <85353442+AnkurChauhan-131@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump github/codeql-action from 2.22.7 to 2.22.8 (#4493)

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 2.22.7 to 2.22.8.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/66b90a5db151a8042fa97405c6cf843bbe433f7b...407ffafae6a767df3e0230c3df91b6443ae8df75)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump actions/setup-java from 3.12.0 to 3.13.0 (#4489)

Bumps [actions/setup-java](https://github.com/actions/setup-java) from 3.12.0 to 3.13.0.
- [Release notes](https://github.com/actions/setup-java/releases)
- [Commits](https://github.com/actions/setup-java/compare/v3.12.0...0ab4596768b603586c0de567f2430c30f5b0d2b0)

---
updated-dependencies:
- dependency-name: actions/setup-java
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump Alfresco/alfresco-build-tools from 4.0.0 to 5.4.0 (#4496)

Bumps [Alfresco/alfresco-build-tools](https://github.com/alfresco/alfresco-build-tools) from 4.0.0 to 5.4.0.
- [Release notes](https://github.com/alfresco/alfresco-build-tools/releases)
- [Commits](https://github.com/alfresco/alfresco-build-tools/compare/8c2a3691aa49cd105f62c2983cda3089b82afe89...042fc28b3e308e1d1159dfee61bc46c5940ed7b6)

---
updated-dependencies:
- dependency-name: Alfresco/alfresco-build-tools
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
